Lib/unittest/mock.py

```diff
@@ -2401,3 +2420,35 @@ def __get__(self, obj, obj_type):
         return self()
     def __set__(self, obj, val):
         self(val)
+
+
+def seal(in_mock):
+    """Disables the automatic generation of "submocks"
+
+    Given an input Mock, seals it to ensure no further mocks will be generated
+    when accessing an attribute that was not already defined.
+
+    Submocks are defined as all mocks which were created DIRECTLY from the
+    parent. If a mock is assigned to an attribute of an existing mock,
+    it is not considered a submock.
+
+    :Example:
+
+    >>> mock = Mock()
+    >>> mock.submock.attribute1 = 2
+    >>> mock.not_submock = mock.Mock()
+    >>> seal(mock)
+    >>> mock.submock.attribute2  # This will raise
+    >>> mock.not_submock.attribute2  # This won't raise
+
```

Comment:
Please move this doc to Doc/library/unittest.mock.rst. Don't forget the ".. versionadded:: 3.7" marker.